hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestObjectGet.java

```diff
@@ -177,33 +177,33 @@ public void getRangeHeader() throws IOException, OS3Exception {
     Response response;
     Mockito.when(headers.getHeaderString(RANGE_HEADER)).thenReturn("bytes=0-0");
 
-    response = rest.get("b1", "key1", 0, null, 0, null);
-    Assert.assertEquals("1", response.getHeaderString("Content-Length"));
-    Assert.assertEquals(String.format("bytes 0-0/%s", CONTENT.length()),
+    response = rest.get("b1", "key1", null, 0, null);
```

Comment:
Curious if it is valid change. I pull the change and saw build failure. Also it is same in master branch
https://github.com/apache/ozone/blob/master/hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestObjectGet.java#L180

Reply:
This is failing in your fork's workflow as well: https://github.com/raju-balpande/apache_ozone/actions/runs/6928699242/job/18844989213

Reply:
I noticed this, there is change in parameters in the called method in parallel. Will address this. thanks.